### PR TITLE
Update cacert.pem (used for validating SSL connections)

### PR DESCRIPTION
**Pull request by mattab**
Used for validating SSL connections within Matomo.

### CODE REVIEW

File: core/DataFiles/cacert.pem [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ## How to update this file
 ## - download latest from https://curl.haxx.se/ca/cacert.pem
 ## - Paste the content below this comment
-## - Leave in there also the Let's encrypt certificate (the first one in the list)
```
Comment (sgiehl) [inline core/DataFiles/cacert.pem]: Was it on purpose to remove the let's encrypt certificate?

Comment (Findus23) [reply core/DataFiles/cacert.pem]: As far as I know the let's encrypt CA is already trusted by every major CA so there should be no reason to additionally add it.

Comment (sgiehl) [reply core/DataFiles/cacert.pem]: If that's the case, it should be good to merge.

Comment (Findus23) [reply core/DataFiles/cacert.pem]: I guess if this breaks anything (which it shouldn't unless something is already broken) is something we will only find out after many users are testing the beta.

Comment (fdellwing) [reply core/DataFiles/cacert.pem]: LE is crosssigned by IdenTrust which is trusted by everything newer than windows XP >SP3.

**Pull request merged**